test/e2e/e2e.go

```diff
@@ -478,10 +482,28 @@ var _ = ginkgo.Describe("e2e control plane", func() {
 			}
 		}
 
-		err := podClient.Delete(context.Background(), podName, metav1.DeleteOptions{})
+		err = podClient.Delete(context.Background(), podName, metav1.DeleteOptions{})
 		framework.ExpectNoError(err, "should delete ovnkube-node pod")
 		framework.Logf("Deleted ovnkube-node %q", podName)
 
+		ginkgo.By("Verifyin that all nodes are schedulable and untainted")
+		err = wait.PollImmediate(2*time.Second, 30*time.Second, func() (bool, error) {
+			afterNodes, err := e2enode.GetReadySchedulableNodes(f.ClientSet)
+			if err != nil {
+				return false, fmt.Errorf("failed to retrieve list of nodes: %w", err)
+			}
+			numBefore := len(beforeNodes.Items)
+			numNow := len(afterNodes.Items)
+			if numNow < numBefore {
+				framework.Logf("Waiting for %d nodes to be fully Ready, Schedulable, and Untainted, only %d are", numBefore, numNow)
+				return false, nil
+			}
+			return true, nil
+		})
+		if err != nil {
```

Comment:
++ nice, let's hope it doesn't get to this point and comes up within the 30 seconds time.